lib/core/src/lib/datatable/components/datatable/datatable.component.html

```diff
@@ -239,8 +243,9 @@
                         (keydown.enter)="onEnterKeyPressed(row, $any($event))"
```

Comment:
Can we replace this label with <th>? It will also resolve this 'minor' issues 
![Screenshot 2025-02-24 at 10 18 20](https://github.com/user-attachments/assets/faeb850b-0005-46b0-8c4f-dcbff95868e7)

Reply:
I am not getting that minor issue on my axe scans. Is there any specific conditions under which it shows up?

Reply:
There is a switch: Best Practices 
![Screenshot 2025-02-24 at 10 18 20](https://github.com/user-attachments/assets/d662d9fb-4554-4bb9-bc4b-42495a2bc3e5)

Reply:
Thanks. Made the suggested change